triggers/daml/Daml/Trigger/Internal.daml

```diff
@@ -32,6 +32,11 @@ import DA.Optional (fromOptional)
 
 import Daml.Trigger.LowLevel hiding (Trigger)
 
+data Version_2_5
+
+version: Version_2_5
+version = error "version should not be called"
```

Comment:
I'd rather avoid `error`, even if we "know" it's not meant to be evaluated. I think something like

```
data Version = Version_2_5
```

should work, we'd just have to use `PackageInterface.lookupDataEnum` instead of `PackageInterface.lookupValue`

Reply:
:+1:

Reply:
Much better with your suggestion. Thanks !